### PR TITLE
Check duplicated classes in dependencies tool

### PR DESCRIPTION
**Pull request by DavidPerezIngeniero**
I've documented the tool in the README.md file.

At first, I wanted to create an independent plugin, but I've seen later, that it shares many code with your plugin, so I've added the functionality.

Basically I've added one task (checkDuplicatedDependencies) and one setting (checkDuplicatedExclude).

### CODE REVIEW

**Comment by DavidPerezIngeniero** [comment]
Sorry, I haven't created a new branch as recommended.

**Comment by DavidPerezIngeniero** [comment]
I suspect that excluded dependencies with the `depend(organization, name)` method are processed.  How can I make them ignored?

**Comment by DavidPerezIngeniero** [comment]
Found this bug: https://github.com/xerial/sbt-pack/issues/66

**Comment by xerial** [comment]
Sorry. I missed this PR. Will review soon.

File: README.md [hunk 1]
```diff
@@ -168,6 +169,92 @@ export PATH=$(HOME)/local/bin:$PATH
 
     $ sbt pack-archive
 
+### Find duplicated classes in dependencies
```
Comment (xerial) [inline README.md]: I feel this explanation is a little bit long for README.md. Could you create a new file (e.g., USAGE.md), then move this section to this file?)

**Comment by xerial** [comment]
Overall, this looks good. Please create USAGE.md for the detailed description of this feature, and add a link to this within README.md

**Comment by DavidPerezIngeniero** [comment]
`README.md` already contains the documentation.

I want to try to improve the performance, by using parallelization.

**Comment by DavidPerezIngeniero** [comment]
OK, I'll create a separate file!

**Comment by DavidPerezIngeniero** [comment]
Done!

Performance has improved a lot by means of parallelization.

**Comment by xerial** [comment]
Merged. Thanks for the contribution!

**Comment by xerial** [comment]
Here is a link to the broken build: https://travis-ci.org/xerial/sbt-pack/builds/58722323

**Comment by DavidPerezIngeniero** [comment]
I'm going to study this problem.  It must be easy to fix.

**Comment by DavidPerezIngeniero** [comment]
It seems that this error is due to the parallelization.  It makes the behaviour random.
It's a pity because it boosts the performance.

**Pull request merged**